chain/chain/src/tests/garbage_collection.rs

```diff
@@ -831,10 +831,9 @@ fn test_clear_old_data_fixed_height() {
 #[test]
 #[allow(unreachable_code)]
 fn ultra_slow_test_clear_old_data_too_many_heights() {
```

Comment:
@pugachAG can we remove this test if it's not working since the last 15 months?

Reply:
yes, I think we can remove it, probably in a separate PR